examples/Cargo.toml

```diff
@@ -21,7 +21,8 @@ members = [
   "trusted_database/module/rust",
   "private_set_intersection/grpc",
   "private_set_intersection/client/rust",
-  "private_set_intersection/module/rust",
+  "private_set_intersection/module_0/rust",
+  "private_set_intersection/module_1/rust",
```

Comment:
Would it make sense (and would it work) to call the modules `main` and `handler` or something more meaningful?